protocol/bin/flow.js

```diff
@@ -119,43 +118,10 @@ function analyzeMessages (json) {
   callback: (null | (err: ?any${r}) => void)
 }`
 
-    return [
-      `// ${json.protocol}.${m} ////////////////////////////////////////`,
-      response, rpc, ''].join('\n\n')
```

Comment:
we're sorting all the things now so splitting up the sections doesn't make sense